dataflows/processors/sort_rows.py

```diff
@@ -1,21 +1,39 @@
+import re
+import decimal
+import datetime
 from kvfile import KVFile
-
 from ..helpers.resource_matcher import ResourceMatcher
 
 
 class KeyCalc(object):
     def __init__(self, key_spec):
         self.key_spec = key_spec
+        self.key_list = re.findall(r'\{(.*?)\}', key_spec)
```

Comment:
I think that a better implementation would be to use a custom Formatter, and add a new 'float type' (or override the existing one) that produces the reversed notation:
https://docs.python.org/3.7/library/string.html#string.Formatter.format_field

wdyt?

Reply:
@roll

Reply:
good idea. let me check